Neos.ContentRepository.LegacyNodeMigration/Classes/Processors/EventExportProcessor.php

```diff
@@ -369,7 +316,7 @@ public function extractPropertyValuesAndReferences(array $nodeDataRow, NodeType
             }
 
             if (!$nodeType->hasProperty($propertyName)) {
-                $this->dispatch(Severity::WARNING, 'Skipped node data processing for the property "%s". The property name is not part of the NodeType schema for the NodeType "%s". (Node: %s)', $propertyName, $nodeType->name->value, $nodeDataRow['identifier']);
+                $context->dispatch(Severity::WARNING, "Skipped node data processing for the property \"{$propertyName}\". The property name is not part of the NodeType schema for the NodeType \"{$nodeType->name->value}\". (Node: {$nodeDataRow['identifier']})");
```

Comment:
i like `sprintf` better ... but ... :D Please dont change it everywhere in Neos 😂